owl/EASE-OBJ.owl

```diff
@@ -993,6 +990,18 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#AestheticDesign -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#AestheticDesign">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Design"/>
+        <rdfs:comment>Aesthetic design *describes* Aesthetics quality of an object.
```

Comment:
I think something is wrong with singular/plural here. one design describes one quality?

Reply:
Well, it can be one to one or one to many relationship between design and quality, if that's what you are asking!

Reply:
No, I think this sentence needs revision.
I mean that it would be rather "aesthetic qualities" or "a aesthetic quality".
- why is aesthetics written capital?
- it is rather aesthetic quality then aesthetics quality, I think
- it is better to not start the comment by repeating the concept name

Better write: *A design that describes an aesthetic quality of an object*